### ISSUE TITLE
warm-up in config setting 

### ISSUE THREAD

**Issue by MahdiMohseni0033** [hailanyi/VirConv#39]
I'm using VirConv-L with the default configuration, but I'm seeing a discrepancy between the warmup period and the actual training process. The config specifies a warmup of 1 epoch, but TensorBoard shows that the learning rate varies over roughly 25 epochs. Am I misunderstanding the config or is there an issue with the implementation? Thank you for your help!
![Screenshot from 2023-08-22 10-16-03](https://github.com/hailanyi/VirConv/assets/108568972/6be7a4b7-5650-4be8-9118-6a188383919d)
